src/filebrowser/crumbs.ts

```diff
@@ -271,7 +263,7 @@ class BreadCrumbs extends Widget {
         if (error.xhr) {
           error.message = `${error.xhr.status}: error.statusText`;
         }
-        if (error.message.indexOf('409') !== -1) {
+        if (error.message.ArrayExt.firstIndexOf('409') !== -1) {
```

Comment:
(There are a few more here, if you search for `.indexOf`, or in the new files, search for `.ArrayExt.firstIndexOf`)